finality-aleph/src/party/member.rs

```diff
@@ -44,3 +59,23 @@ pub fn task<
     let handle = spawn_handle.spawn_essential("aleph/consensus_session_member", task);
     Task::new(handle, stop)
 }
+
+fn rotate_saved_unit_files(stash_path: &Path, session_id: u32) -> Result<(File, File), io::Error> {
+    let extension = ".abfst";
+    let session_path = stash_path.join(format!("{}", session_id));
+    fs::create_dir_all(&session_path)?;
+    let index = fs::read_dir(&session_path)
+        .unwrap()
```

Comment:
`?` won't work?

Reply:
`?` should work fine. This particular function just punts any I/O errors back up to `task()`.

Reply:
does it? all other io-related things here will result in `Err(_)`, but this one will just `panic` - am I right?